lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && err) {
```

Comment:
in neither of those situations will the t.end be called with any arguments though.